src/local_wifi_password.h

```diff
@@ -0,0 +1,6 @@
+// Modify this file on your local.
+
+// Tell git to ignore it's changes so you don't commit your credentials
+// git update-index --assume-unchanged local_wifi_password.h

```

Comment:
I shall have to remember this trick as it's a situation I've encountered before and I guessed there must be some way to make git ignore changes to the file even though the base/generic version has been added to the repo.